cluster/environments/azure-common-infra/backend.tfvars

```diff
@@ -1,4 +1,5 @@
-storage_account_name="<storage account name>"
+/*storage_account_name="<storage account name>"
```

Comment:
These don't need to be commented out, if disabling backend storage, then comment out the appropriate `backend "azurerm" {}` line.

@timfpark - I know we want backend state enabled, for tests, that would require the creation of a storage account to do so and integration into the test bed.  Should this be the casE?